src/notebook-jobs-panel.tsx

```diff
@@ -90,68 +93,82 @@ export class NotebookJobsPanel extends VDomRenderer<JobsModel> {
       this.model.jobsView = JobsView.CreateForm;
     };
 
+    const trans = this._translator.load('jupyterlab');
+
```

Comment:
This can be defined in `constructor()` and bound to an instance variable instead of loading this every render.